### PR TITLE
Update ZH

### PR DESCRIPTION
**Pull request by nLight**
`Applied` is not deleted, just moved.

### CODE REVIEW

**Comment by pierrebeitz** [comment]
key `Mesos DNS` is missing...

**Comment by pierrebeitz** [comment]
also the file does not seem to be valid JSON anymore... fixing...

**Comment by GeorgiSTodorov** [comment]
@pierrebeitz , I just pushed.

**Comment by pierrebeitz** [comment]
@GeorgiSTodorov you lost "Partially applied". force-pushing.

File: locale/zh/messages.json [hunk 1]
```diff
@@ -984,49 +984,49 @@
...
   "your service's role ({requestedValue}).": "您服务的角色 ({requestedValue})。",
-  "{0, plural, one {# group has services not limited by quota. Update service roles to have quota enforced.} other {# groups have services not limited by quota. Update service roles to have quota enforced.}}": "",
-  "{0, plural, one {# instance running out of {instancesCount}} other {# instances running out of {instancesCount}}}": "",
-  "{0, plural, one {# service has no quota limit.} other {# services have no quota limit.}}": "",
-  "{0}": "",
+  "{0, plural, one {# group has services not limited by quota. Update service roles to have quota enforced.} other {# groups have services not limited by quota. Update service roles to have quota enforced.}}": "{0, plural, one {# 组的服务不受配额限制。更新服务角色以执行配额。} other {# 组的服务不受配额限制。更新服务角色以执行配额。}}",
+  "{0, plural, one {# instance running out of {instancesCount}} other {# instances running out of {instancesCount}}}": "{0, plural, one {# 实例将用完 {instancesCount}} other {# 实例将用完 {instancesCount}}}",
+  "{0, plural, one {# service has no quota limit.} other {# services have no quota limit.}}": "{0, plural, one {# 服务没有配额限制。} other {# 服务没有配额限制。}}",
+  "{0}": "{0}",
```
Comment (pierrebeitz) [inline locale/zh/messages.json]: what the heck? looks like someone put a variable in a `Trans` that has not been `i18nMark`ed which is likely a mistake.

**Comment by GeorgiSTodorov** [comment]
Maybe we should remove the check.

**Comment by pierrebeitz** [comment]
the critical part of the build-stage is done. merging.

File: locale/zh/messages.json (hunk 1 above)
Comment (TattdCodeMonkey) [reply locale/zh/messages.json]: Yea, someone i18nMarked or trans a variable. which will never actually translate anything.

**Pull request merged**